examples/using-preact/package.json

```diff
@@ -9,7 +9,9 @@
   "dependencies": {
     "next": "^2.0.0-beta",
     "preact": "^7.1.0",
-    "preact-compat": "^3.9.4"
+    "preact-compat": "^3.9.4",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
```

Comment:
we don't *need* it, but shrinkwrap will error out with some awkward errors due to missing deps